staging/src/k8s.io/cluster-bootstrap/jws/jws.go

```diff
@@ -23,10 +23,10 @@ import (
 	jose "gopkg.in/square/go-jose.v2"
 )
 
-// computeDetachedSig takes content and token details and computes a detached
+// ComputeDetachedSig takes content and token details and computes a detached
 // JWS signature.  This is described in Appendix F of RFC 7515.  Basically, this
 // is a regular JWS with the content part of the signature elided.
-func computeDetachedSig(content, tokenID, tokenSecret string) (string, error) {
+func ComputeDetachedSig(content, tokenID, tokenSecret string) (string, error) {
```

Comment:
if we're exporting this, name it `ComputeDetachedSignature`

Reply:
+1 on the name change